addon/mixins/route.js

```diff
@@ -199,6 +200,17 @@ export default Ember.Mixin.create({
   },
 
   /**
+   Call additional functions after finding the infinityModel in the Ember data store.
+
+   @method afterInfinityModel
+   @param {Function} infinityModelPromise The resolved result of the Ember store find method. Passed in automatically.
+   @return {Ember.RSVP.Promise}
+   */
+  afterInfinityModel(infinityModelPromiseResult) {
+    return infinityModelPromiseResult;
```

Comment:
Provided the user returns something that can `get('content')` we should be fine yeah?